src/pages/components/datepicker.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Datepicker
+description: >
+  A Datepicker lets users select a date by showing them a calendar. It works in combination with a text input.
+package: '@zendeskgarden/react-datepickers'
+components:
+  - datepickers/src/elements/Datepicker/Datepicker.tsx
+  - datepickers/src/elements/DatepickerRange/DatepickerRange.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To select a single date, use a Datepicker</li>
+      <li>To select a start and an end date, use a Datepicker range</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Datepicker component.
+
+import Default from '../../examples/datepicker/DatepickerDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/datepicker/DatepickerDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Custom date format
+
+Dates are formatted using the
+[Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat)
+utility. For other formats, consider libraries like
+[momentjs](https://momentjs.com/) or [date-fns](https://date-fns.org/).
+
+import Custom from '../../examples/datepicker/DatepickerCustom.tsx';
+import CustomCode from '!!raw-loader!../../examples/datepicker/DatepickerCustom.tsx';
+
+<CodeExample code={CustomCode}>
+  <Custom />
+</CodeExample>
+
+### Datepicker range
+
+Datepicker range works in combination with two text inputs. It displays a
+calendar that lets users choose a start and end date.
+
+import Range from '../../examples/datepicker/DatepickerRange.tsx';
+import RangeCode from '!!raw-loader!../../examples/datepicker/DatepickerRange.tsx';
+
+<CodeExample code={RangeCode}>
+  <Range />
+</CodeExample>
+
+### Localization
+
+Dates are localized using the
+[Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat)
+utility. For other formats, use `customParseDate` prop, and formatting libraries
+like [momentjs](https://momentjs.com/) or [date-fns](https://date-fns.org/).
+Customize language, region, and calendar type with a [BCP 47 language
+tag](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat#Parameters).
```

Comment:
This sentence isn't harmonizing nicely with the provided example. "BCP 47 language tag" !== `locale` prop, causing confusion. I'd recommend removing altogether, allowing the example to stand on its own. Someone can learn more about the `locale` prop by visiting the API section below where there is a link to help learn about the "Unicode BCP 47 locale identifier."